workflow/scripts/build_existing_chp_de.py

```diff
@@ -16,31 +16,25 @@
 import sys
 
 
-def clean_data(combustion, biomass, geodata, opts):
+def clean_data(combustion, biomass, geodata):
     """
     Clean the data and return a dataframe with the relevant information.
     PLZ is translated to longitude and latitude using the pyGeoDb data.
     """
     biomass.dropna(subset="Postleitzahl", inplace=True)
     biomass.rename(columns={'NameStromerzeugungseinheit': 'NameKraftwerk'}, inplace=True)
-
-    if 'I' not in opts:
-        # filter out industrial CHPs if industry is not included
-        industry = combustion.dropna(subset='Einsatzort')
-        to_drop = industry[industry.Einsatzort.str.contains('Industrie')].index
-        combustion.drop(index=to_drop, inplace=True)
+    biomass['Einsatzort'] = ''
 
     data = pd.concat([biomass, combustion], join='inner', ignore_index=True)
-
+    
+    data['IndustryStatus'] = data['Einsatzort'].str.contains('Industrie')
+    data['IndustryStatus'] = data['IndustryStatus'].fillna(False)
```

Comment:
```suggestion
    data['IndustryStatus'] = data['Einsatzort'].str.contains('Industrie').fillna(False)
```
Possibly one can even drop the fillna, because I think `.str` accessor will complain about NaNs